lib/ansible/modules/cloud/amazon/s3_lifecycle.py

```diff
@@ -68,6 +102,14 @@
   transition_days:
     description:
       - "Indicates when, in days, an object transitions to a different storage class. If transition_date is not specified, this parameter is required."
+  transitions:
+    description:
+      - A list of transition behaviors to be applied to the rule. Each storage class may be used only once. Each transition
+        behavior contains these elements
```

Comment:
s/contains/may contain/? Initially I read this to mean `contains all three elements`